tests/queries/0_stateless/02699_polygons_sym_difference_rollup.reference

```diff
@@ -2,6 +2,8 @@
 []
 [[(2147483647,0),(10.0001,65535),(1,255),(1023,2147483646)]]	[[[(2147483647,0),(10.0001,65535),(1023,2147483646),(2147483647,0)]]]
 [[(2147483647,0),(10.0001,65535),(1,255),(1023,2147483646)]]	[]
+[[(2147483647,0),(10.0001,65535),(1,255),(1023,2147483646)]]	[[[(2147483647,0),(10.0001,65535),(1023,2147483646),(2147483647,0)]]]
```

Comment:
we got a different result with the new analyser, is it ok?

Reply:
It's about WITH TOTALS and constants. In the previous analyzer, TOTALS pocket was sent with the default value for the corresponding type instead of constant. This behavior changed in the new analyzer and we don't think that's a problem because it wasn't specified anyhow.